src/LinkedList.js

```diff
@@ -0,0 +1,129 @@
+class Node {
+  constructor() {
+    this.data
+    this.next = null
+  }
+}
+
+class LinkedList {
+  constructor() {
+    this.head = null
+    this.tail = null
+  }
+
+  getHeadNode() {
+    return this.head
+  }
+
+  getTailNode() {
+    return this.tail
+  }
+
+  contains(element) {
+    let elementFound = false
+    let iterator = this.head
```

Comment:
I'd name this `currentNode`, because `iterator` is a special term to describe an object that accesses elements in a collection one at a time.

You can read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Iterators_and_Generators).